catalog.html

```diff
@@ -0,0 +1,1394 @@
+          <img
+            src="./images/icons/logo.svg"
+            alt="Pixmart logo"
+            width="108"
+            height="24"
+          />
+        </strong>
+      </a>
+      <nav class="header__navigation">
+        <ul class="navigation__list list">
+          <li class="navigation__list-item">
+            <a
+              href="#"
+              class="navigation__link button button__small button__navigation"
+            >
+              About
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="./catalog.html"
+              class="navigation__link button button__small button__navigation"
+            >
+              Products
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="#"
+              class="navigation__link button button__small button__navigation"
+            >
+              Creators
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="./blog-details.html"
+              class="navigation__link button button__small button__navigation"
+            >
+              Blog
+            </a>
+          </li>
+          <li class="navigation__list-item">
+            <a
+              href="./contacts.html"
+              class="navigation__link button button__small button__navigation"
+            >
+              Contact
+            </a>
+          </li>
+        </ul>
+      </nav>
+      <div class="header__assets-block list">
+        <form action="#" method="get" class="assets-block__searching">
+          <label for="asset-searching" class="searching__label hidden"
+            >Find asset</label
+          >
+          <input
+            type="text"
+            class="searching__input input input__small"
+            id="asset-searching"
+            placeholder="Search for Digital Assets"
+          />
+          <button type="submit" class="searching__button hidden">Search</button>
+        </form>
+        <a
+          href="#"
+          class="assets-block__button button button__primary button__square-medium"
+        >
+          <svg class="assets-block__icon icon">
+            <use xlink:href="./images/icons/sprite.svg#bag"></use>
+          </svg>
+        </a>
+      </div>
+    </header>
+    <main class="container__main">
+      <div class="introduction container__section">
+        <h1 class="introduction__semantic-title hidden">Catalog</h1>
+        <strong
+          class="introduction__title heading__main-title main-information__highlite-text"
+        >
+          Explore Top
+          <span class="highlite-text__span">UI Kits</span>&nbsp;Design Assets
+        </strong>
+      </div>
+      <div class="container__catalog container__section list">
+        <aside class="sidebar">
+          <form action="#" method="get" class="filter-form">
+            <div class="filter-form__container">
```

Comment:
The [fieldset](https://html.spec.whatwg.org/multipage/form-elements.html#the-fieldset-element) element [represents](https://html.spec.whatwg.org/multipage/dom.html#represents) a set of form controls (or other content) grouped together, optionally with a caption. The caption is given by the first [legend](https://html.spec.whatwg.org/multipage/form-elements.html#the-legend-element) element that is a child of the [fieldset](https://html.spec.whatwg.org/multipage/form-elements.html#the-fieldset-element) element, if any. The remainder of the descendants form the group.

Reply:
Great job Alina, almost done!
But one little thing - you don't need to put headings(H1-H2-H3...) in Legend tag, because Legend tag represents the group name.

So you can easily transform it into:

`<legend>
                <h2 class="filter-form__title heading__form">Categories</h2>
              </legend>`

==>

`<legend>Categories</legend>`

Reply:
Fixed 🥇

Reply:
Not fixed

Reply:
Fixed 🥇